config/locales/en/course/assessment/question/text_response.yml

```diff
@@ -7,9 +7,6 @@ en:
             text_response: 'New Text Response Question'
             file_upload: 'New File Upload Question'
             comprehension: 'New Comprehension Question'
-          solution_fields:
```

Comment:
Port this as well? Please be careful when porting pages. Ensure we have ported everything before deleting anything.